frontend/packages/kubevirt-plugin/integration-tests/v2v/rhv/v2v.rhv.import.negative.scenario.incorrect.names.ts

```diff
@@ -0,0 +1,72 @@
+import { browser } from 'protractor';
+import { asyncForEach, click, waitForStringInElement } from '@console/shared/src/test-utils/utils';
+import { VirtualMachineModel } from '@console/kubevirt-plugin/src/models';
+import { CHARACTERS_NOT_ALLOWED, SEC } from '../../tests/utils/constants/common';
+import { RhvImportWizard } from '../../tests/models/rhvImportWizard';
+import * as rhvView from '../../views/rhvImportWizard.view';
+import * as view from '../../views/importWizard.view';
+import { rhvVMConfigNoStartOnCreate } from './v2v.rhv.configs';
+import { getWrongDisk, getWrongNic } from '../../tests/utils/utils';
+import { wrongValues } from '../../tests/utils/constants/vm';
+
+const { provider, instanceConfig, name, sourceVMName } = rhvVMConfigNoStartOnCreate;
+
+describe('VMWare Wizard validation, negative tests', () => {
+  const wizard = new RhvImportWizard();
+
+  beforeEach(async () => {
+    await wizard.openWizard(VirtualMachineModel);
+  });
+
+  afterEach(async () => {
+    await wizard.closeWizard();
+  });
+
+  async function startWizardTillName() {
```

Comment:
Maybe `processInstanceConnection` would be a more suitable name?

Reply:
Done